imap/jmap_mail.c

```diff
@@ -10908,6 +10908,9 @@ static int _email_mboxrecs_read_cb(const conv_guidrec_t *rec, void *_rock)
     /* don't process emails that have this email attached! */
     if (rec->part) goto done;
 
+    /* don't process emails that have been expunged */
+    if (rec->internal_flags & FLAG_INTERNAL_EXPUNGED) goto done;
```

Comment:
Also ignore if FLAG_DELETED is set in the system_flags?

Reply:
Agreed